msvc/sdk-build.props

```diff
@@ -24,6 +24,10 @@
     </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Condition="'$(BuildingFrameworksCore)' != 'true'" Include="WinObjC.Frameworks.UWP" Version="*" />
```

Comment:
That doesn't mean it can't go in the same block of package references... that just means we have to keep the Condition